translating-guide/en/managing-languages.md

```diff
@@ -65,3 +65,35 @@ If you don't see the language(s) you are looking for listed on either of these l
 ## Install a Language
 
 Site Administrators can install new languages through the Administration dashboard. See [Learning OJS 3](/learning-ojs/en/site-administration#languages) for more information on how to install and manage languages in PKP software.
+
+If you have newly translated the software into another language and want to install that language on your site, you will not be able to install it in the standard way until it has been merged into a release of the software. Instead, if you have access to the server and installation files and sufficient technical knowledge, you can install the new language using one of the methods below. 
+
+First, confirm that you are using OJS/OMP/OPS 3.2.0 or above. If you are not, you will need to upgrade to the newest version of OJS/OMP/OPS.
+
+When you upgrade, if you use a git checkout of OJS/OMP/OPS, rather than installing from a .tar.gz package, you will install the new translation files.
```

Comment:
I'm not sure where this comes from -- maybe it should be:
> When you upgrade, if you use a git checkout of OJS/OMP/OPS, rather than installing from a .tar.gz package, you will use the latest "stable" branch of the software. This will include recent translation contributions automatically and can be periodically refreshed.

Reply:
I think I extrapolated this from some brief instructions you wrote on the forum. Thanks for clarifying exactly how it works.